updater/openmaptiles.patch

```diff
@@ -10,6 +10,15 @@ index 5ce2248..6b63487 100644
  
  build/mapping.yaml: init-dirs
  	$(DOCKER_COMPOSE) run $(DC_OPTS) openmaptiles-tools generate-imposm3 openmaptiles.yaml > $@
+@@ -280,6 +281,8 @@ bash:
+ .PHONY: import-wikidata
+ import-wikidata:
+ 	$(DOCKER_COMPOSE) run $(DC_OPTS) openmaptiles-tools import-wikidata --cache /cache/wikidata-cache.json openmaptiles.yaml
++	$(DOCKER_COMPOSE) run $(DC_OPTS) openmaptiles-tools psql.sh -v ON_ERROR_STOP=1 -P pager=off \
++	    -c "UPDATE wd_names SET labels = labels || hstore('name:be-tarask', REGEXP_REPLACE(labels->'name:be-tarask', ' *\(.+\) *', '') ) WHERE labels->'name:be-tarask' LIKE '%(%)%';"
```

Comment:
Why not to land it upstream?

Reply:
want to get more feedback, @Komzpa is this PR looks good to be merged ? Can you give advice were nice to deploy this changes (old or new server)?